src/constants/sizes.js

```diff
@@ -6,6 +6,12 @@ export const SIZES = Object.freeze({
   LARGE: "large"
 });
 
+export const SIZES_BASIC = Object.freeze({
```

Comment:
That's great!
but maybe let's move the SIZES_BASIC to be before the SIZES init and do something like this:
SIZES = Object.freeze({
  XXS: "xxs",
  XS: "xs",
  SMALL: "small",
...SIZES_BASIC
});

Reply:
Done!